pkg/controller/datadogagentdeployment/utils.go

```diff
@@ -635,6 +662,10 @@ func getVolumeMountsForAgent(spec *datadoghqv1alpha1.DatadogAgentDeploymentSpec)
 			Name:      datadoghqv1alpha1.ConfdVolumeName,
 			MountPath: datadoghqv1alpha1.ConfdVolumePath,
 		},
+		{
+			Name:      datadoghqv1alpha1.ChecksdVolumeName,
+			MountPath: datadoghqv1alpha1.ChecksdVolumePath,
```

Comment:
we might want to add `ReadOnly:  true,` here - same for `confd`
( [helm chart](https://github.com/helm/charts/blob/master/stable/datadog/templates/container-agents.yaml#L170) )

Reply:
Good point, thank you! Will update to be `ReadOnly`.